backend/typescript/migrations/20230213004442-update-creator-model.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
```

Comment:
So, it looks like this just generates an empty migration file. Looking further into it, sequelize doesn't automatically generate migrations... unlike django's ORM or sqlalchemy or literally anything else...

Apparently the best way to do this is to just generate the model again using the command described here: https://sequelize.org/docs/v6/other-topics/migrations/#creating-the-first-model-and-migration . Look at the types defined in the model file and used them here. This should recreate the model file.